gdrive/sheets/types.py

```diff
@@ -0,0 +1,98 @@
+from abc import ABC, abstractmethod
+from enum import Enum
+
+
+class FilterTypeEnum(str, Enum):
+    TEXT_CONTAINS = "TEXT_CONTAINS"
+    TEXT_EQUALS = "TEXT_EQ"
+    CUSTOM = "CUSTOM_FORMULA"
+
+
+class SortOrderEnum(str, Enum):
+    ASCENDING = "ASCENDING"
+    DESCENDING = "DESCENDING"
+
+
+class SummarizeFunctionEnum(str, Enum):
+    SUM = "SUM"
+
+
+class ValueLayoutEnum(str, Enum):
+    HORIZONTAL = "HORIZONTAL"
+
+
+class FormulaEnum(str, Enum):
+    NOOP = ""
+    OR = "OR"
+    SUM = "sum"
+    REGEX_MATCH = "regexmatch"
+    GET_PIVOT_DATA = "GETPIVOTDATA"
+
+
+class StringLiteral:
```

Comment:
what does this do?

Reply:
I should document these types.

Reply:
StringLiteral is used in the formula builder to denote a value that should be rendered as a string in the formula rather than a token